Ssl

matrinox opened the pull request.

geoff-lee-lendesk committed:
Add socket_opts from config

Reorganize & refactor code; redis_opts returns only redis options and the rest are returned from connection_opts
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
On branch ssl - Thu 31 Aug 2017 23:11:57 PDT by Geoff Lee <geoff.lee@lendesk.com>

Fix tests

Add test for Exq.Support.Opts.connection_opts
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
On branch ssl - Tue  5 Sep 2017 15:14:25 PDT by Geoff Lee <geoff.lee@lendesk.com>
Amended commit - Tue  5 Sep 2017 15:21:13 PDT by Geoff Lee <geoff.lee@lendesk.com>

senorprogrammer reviewed (commented):
I'm not qualified to comment on this project or these changes, I don't know enough about them, or the impact these will have on this codebase.

Seems like something that should be opened as a PR against the parent project and have the maintainers look at?

AaronBetterOfficeApps commented on mix.exs:
```diff
@@ -35,7 +35,7 @@ defmodule Exq.Mixfile do
     [
       { :uuid, ">= 1.1.0" },
-      { :redix, ">= 0.5.0"},
+      { :redix, ">= 0.6.1" },
```
Is this our version of remix? if so I think we should add something like `0.6.1-ld` so that it won't conflict with the office `0.6.1`

matrinox replied:
They reference the commit sha so it will match the correct version. Are you saying it's a conflict in reading it?

matrinox commented:
@senorprogrammer done https://github.com/akira/exq/pull/285

geoff-lee-lendesk committed:
Remove OTP 17.5

Not supported by elixir versions 1.3.2 or 1.4.0 (see travis logs)
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
On branch ssl - Tue  5 Sep 2017 15:25:14 PDT by Geoff Lee <geoff.lee@lendesk.com>
